### DISCUSSION TITLE
Plan to merge SYCL into LLVM upstream?

### DISCUSSION THREAD

**Discussion by cdluminate** [intel/llvm#3250] [General]
Hi! Do you have any plan to merge SYCL into LLVM upstream? I'm working on Debian packaging. Upstreaming SYCL into LLVM upstream may enable me to start working on SYCL-related packages without introducing a bunch of different LLVMs into the distro.

**Comment by MrSidims** [comment]
Hi, yes there are plans and some work is in progress: https://github.com/intel/llvm/projects/1 . Unsure how much effort has left, @bader and @Fznamznon shall know better.
